### PR TITLE
Start using Niche targeting as a marketing term

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: content/pages/similar-pages.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Title: Automatic Campaign Targeting
+Title: Niche Campaign Targeting
```
Comment (davidfischer) [inline content/pages/similar-pages.md:1]:
```suggestion
Title: Niche Contextual Targeting
```

Should we take Greg's feedback and also use contextual targeting here?

**Review by davidfischer — APPROVED** [review]
Should we also link from this to https://github.com/readthedocs/ethicalads.io/pull/426?

File: content/pages/similar-pages.md (hunk 1 above)
Comment (ericholscher) [reply content/pages/similar-pages.md:1]: Sure 👍

**Comment by ericholscher** [comment]
Yea, I can add the link in that post.

**Pull request merged**